misc/wsdl-to-ballerina/src/main/java/io/ballerina/wsdltoballerina/GeneratedSourceFile.java

```diff
@@ -0,0 +1,45 @@
+/*
+ *  Copyright (c) 2024, WSO2 LLC. (http://www.wso2.com)
+ *
+ *  WSO2 LLC. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ */
+
+package io.ballerina.wsdltoballerina;
+
+public class GeneratedSourceFile {
+    private String fileName;
+    private String content;
+
+    public GeneratedSourceFile(String fileName, String content) {
```

Comment:
Since we are setting the values for `fileName` and `content` field in the constructor itself, do we need the setters ? If there is no requirement for setters we can mark `fileName` and `content` fields as `final` as well.